### PR TITLE
correcting references to api_key

### PR DESCRIPTION
**Pull request by ahmadnassri**
checking [travis-ci/dpl](https://github.com/travis-ci/dpl) its clear that the proper key should be `api_key`

### CODE REVIEW

**Comment by plaindocs** [comment]
I agree that consistency is a good thing. If you feel like changing them all to `api-key` as @rkh prefers I'll merge this in.

**Comment by rkh** [comment]
I prefer `_` in YAML, `-` for command line arguments, as that makes them feel natural to me.

File: user/deployment/cloudfiles.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ This example is almost certainly not ideal, as you probably want to upload your
 It is recommended encrypt that you encrypt your Rackspace api key.
 Assuming you have the Travis CI command line client installed, you can do it like this:
 
-    travis encrypt --add deploy.api-key
+    travis encrypt --add deploy.api_key
```
Comment (plaindocs) [inline user/deployment/cloudfiles.md]: This looks like the only CLI one of the lot.

**Comment by plaindocs** [comment]
Thanks for the clarification @rkh

**Comment by ahmadnassri** [comment]
folks, it seems I have removed the original fork, if you're interested in me correcting that one `-` in the cli code as mentioned here, happy to start a new pull request with the same changes ... 

let me know.

**Comment by plaindocs** [comment]
Please do, I'll make sure we merge it straight in this time.

**Comment by ahmadnassri** [comment]
done in #550

**Pull request closed without merging**